easybuild/easyconfigs/n/NextClip/NextClip_scripts_shebang.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/scripts/index_bwa.sh b/scripts/index_bwa.sh
```

Comment:
@nathanhaigh: please add a line of comment above this line describing what this patch does, and add yourself as patch author


```
Fix hardcoding of `/usr/bin/perl`, use `perl` available through `$PATH`
author: Nathan Haigh (<affiliation>)
```